improver_tests/psychrometric_calculations/total_precipitable_water/test_total_precipitable_water.py

```diff
@@ -0,0 +1,99 @@
+# (C) Crown Copyright, Met Office. All rights reserved.
+#
+# This file is part of 'IMPROVER' and is released under the BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+"""Unit tests for the PrecipitableWater plugin"""
+
+import iris
+import iris.coords
+import iris.cube
+import numpy as np
+import pytest
+
+from improver.psychrometric_calculations.total_precipitable_water import (
+    PrecipitableWater,
+)
+from improver.synthetic_data.set_up_test_cubes import set_up_variable_cube
+
+
+@pytest.fixture
+def mock_cube():
+    """
+    Fixture to create a mock humidity mixing ratio cube with pressure levels.
+    """
+
+    pressure_levels = np.array(
+        [100000, 97500, 95000, 92500, 90000, 85000, 80000, 75000, 70000],
+        dtype=np.float32,
+    )  # Pa
+    data = np.random.uniform(0.001, 0.01, size=(len(pressure_levels), 5, 5)).astype(
+        np.float32
+    )  # realistic variation
+
+    # Create a basic cube
+    cube = set_up_variable_cube(
+        data, name="specific_humidity", units="kg kg-1", spatial_grid="latlon"
+    )
+
+    # Replace the vertical coordinate with pressure
+    pressure_coord = iris.coords.DimCoord(
+        pressure_levels,
+        units="Pa",
+        long_name="pressure",
+        var_name="pressure",
+    )
+    cube.remove_coord("realization")  # if present
+    cube.add_dim_coord(pressure_coord, 0)
+
+    # Add title attribute
+    cube.attributes["title"] = (
+        "Global Enhanced Model Forecast on Global 10 km Standard Grid"
+    )
```

Comment:
This gives the same result:
```suggestion
    cube = set_up_variable_cube(
        data, name="specific_humidity", units="kg kg-1", spatial_grid="latlon",
        vertical_levels=pressure_levels, pressure=True,
        attributes={"title": "Global Enhanced Model Forecast on Global 10 km Standard Grid"}
    )
```